src/os/Gpio.hxx

```diff
@@ -37,6 +37,13 @@
 
 #include "utils/macros.h"
 
+#if defined(ESP32)
+#undef LOW
```

Comment:
Hmm...  These are unfortunate.  Should we change the OpenMRN side?

Reply:
Perhaps, it is the same issue as observed in DISPLAY. It isn't really being done correctly on the arduino-esp32 side (or ArduinoAvr-Core for that matter likely)